app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
+  $studio_due = Time.parse("2016/01/10 23:59")
```

Comment:
I set this to global variable so it can be accessed anywhere throughout application.
I assumed the deadline to sign up for studio this time is Jan 10, 2016 (subject-to-change later).